scripts/analyze.sh

```diff
@@ -314,8 +314,10 @@ if check_for_require ocf || check_config_file ZJS_OCF; then
 fi
 
 if check_for_require gpio || check_config_file ZJS_GPIO; then
+    >&2 echo Using module: GPIO
     MODULES+=" -DBUILD_MODULE_GPIO"
     echo "CONFIG_GPIO=y" >> $PRJFILE
+    echo "export ZJS_BOARD=y" >> $CONFFILE
```

Comment:
It shouldn't hurt anything and it is needed. If they require('board') they need board; if they require('gpio') they need board even if they didn't require('board'), but they don't need the JS board API part. This will need some rework at some point, perhaps the pin stuff should be separated from the board module. The board has become utterly trivial now, basically just a name. When I started this patch it was much bigger. :)